diff -urN libvpx-1.5.0.orig//build/make/gen_msvs_vcxproj.sh libvpx-1.5.0/build/make/gen_msvs_vcxproj.sh
--- libvpx-1.5.0.orig//build/make/gen_msvs_vcxproj.sh	2015-11-09 22:12:38 +0000
+++ libvpx-1.5.0/build/make/gen_msvs_vcxproj.sh	2016-01-13 15:37:29 +0000
@@ -295,17 +295,13 @@
         tag_content ProjectGuid "{${guid}}"
         tag_content RootNamespace ${name}
         tag_content Keyword ManagedCProj
-        if [ $vs_ver -ge 12 ] && [ "${platforms[0]}" = "ARM" ]; then
+        if [ $vs_ver -ge 12 ]; then
+            tag_content DefaultLanguage "en-US"
             tag_content AppContainerApplication true
-            # The application type can be one of "Windows Store",
-            # "Windows Phone" or "Windows Phone Silverlight". The
-            # actual value doesn't matter from the libvpx point of view,
-            # since a static library built for one works on the others.
-            # The PlatformToolset field needs to be set in sync with this;
-            # for Windows Store and Windows Phone Silverlight it should be
-            # v120 while it should be v120_wp81 if the type is Windows Phone.
             tag_content ApplicationType "Windows Store"
-            tag_content ApplicationTypeRevision 8.1
+            tag_content ApplicationTypeRevision 10
+            tag_content WindowsTargetPlatformVersion 10.0.10240.0
+            tag_content WindowsTargetPlatformMinVersion 10.0.10069.0
         fi
     close_tag PropertyGroup
 
